libbeat/beat/beat.go

```diff
@@ -28,14 +28,13 @@ import (
 	"runtime"
 	"sync"
 
-	"github.com/satori/go.uuid"
-	"github.com/urso/ucfg"
-
 	"github.com/elastic/beats/libbeat/cfgfile"
+	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/filter"
 	"github.com/elastic/beats/libbeat/logp"
 	"github.com/elastic/beats/libbeat/publisher"
 	"github.com/elastic/beats/libbeat/service"
+	"github.com/satori/go.uuid"
```

Comment:
me too, but for some reason there is a problem with github.com/satori/go.uuid . I'm happy it's not automatically removed anymore (we have had PR's removing import of this lib in past due to goimports)